config/locales/en/providers.yml

```diff
@@ -1001,6 +1001,7 @@ en:
           multiple_employments: Multiple employments
           refunds: Tax or NI refunds
           uploaded_bank_statements: Bank statements uploaded
+          ineligible: Ineligible
```

Comment:
Is this the value that is added to the means report for caseworker review?
I _think_ it is supposed to say `Applicant is ineligible` but you may need to chat with Dave to confirm

Reply:
👍 Asked Dave and he said if he had the choice and it is easy to amend then 'Applicant is ineligible' is probably clearer so have amended the value